Orange/widgets/data/owselectrows.py

```diff
@@ -628,20 +626,33 @@ def commit(self):
         self.match_desc = report.describe_data_brief(matching_output)
         self.nonmatch_desc = report.describe_data_brief(non_matching_output)
 
-        self.update_info(matching_output, self.data_out_rows, "Out: ")
+        self.update_output_summary(matching_output)
 
-    def update_info(self, data, lab1, label):
+    def update_input_summary(self, data):
         def sp(s, capitalize=True):
             return s and s or ("No" if capitalize else "no"), "s" * (s != 1)
 
-        if data is None:
-            lab1.setText("")
+        if data:
+            n = data.approx_len()
+            details = "~%s row%s, %s variable%s" % \
+                      (sp(n) + sp(len(data.domain.variables) + len(data.domain.metas)))
```

Comment:
Try to keep the line length up to 79 characters.

Reply:
Also, consider replacing formatting with `%` with formatting with f-strings.